Add support for AWS account aliases (#1434)
nerdswords/yet-another-cloudwatch-exporter: README.md

"shield:ListProtections"
```

> This permission is required to get the AWS account alias, used in the the `aws_account_info` metric

Reviewer: maybe useful to include a link to https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html?

```suggestion
This permission is required to get the [AWS account alias](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html), used in the the `aws_account_info` metric
```

Author: did a mix of both

> AWS supports creating something called AWS account aliases. This are human friendly name that can be used to easily identify accounts. An account can have at most a single account alias ([docs](https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccountAliases.html)), and aliases are unique across an AWS network partition ([docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#AboutAccountAlias)). The follwoing permission is required to get the AWS account alias, used in the the `aws_account_info` metric

Reviewer: ```suggestion
The AWS IAM API supports creating Account Aliases, which are human-friendly names that can be used to easily identify accounts. An account can have at most a single account alias ([docs](https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccountAliases.html)), and aliases are unique across an AWS network partition ([docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#AboutAccountAlias)). The following permission is required to get the AWS account alias, used in the the `aws_account_info` metric:
```
